cloud-core/src/main/java/cloud/commandframework/Command.java

```diff
@@ -358,17 +358,18 @@ public Command(
     /**
      * Get the description for an argument
      *
+     * @param sender Command sender
      * @param argument Argument
      * @return Argument description
      * @throws IllegalArgumentException If the command argument does not exist
      * @deprecated More than one matching command argument may exist per command.
      *             Use {@link #getArguments()} and search in that, instead.
      */
     @Deprecated
-    public @NonNull String getArgumentDescription(final @NonNull CommandArgument<C, ?> argument) {
+    public @NonNull String getArgumentDescription(final C sender, final @NonNull CommandArgument<C, ?> argument) {
```

Comment:
api break

Reply:
How would you want to handle this case? If the sender is not provided, then would you call the new method with sender with a null value, or would you make up a new sender object that does nothing?